site/src/pages/ManagementSettingsPage/IdpSyncPage/IdpSyncPageView.tsx

```diff
@@ -0,0 +1,269 @@
+import type { Interpolation, Theme } from "@emotion/react";
+import { useTheme } from "@emotion/react";
+import LaunchOutlined from "@mui/icons-material/LaunchOutlined";
+import Button from "@mui/material/Button";
+import Skeleton from "@mui/material/Skeleton";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import type { OIDCConfig } from "api/typesGenerated";
+import { ChooseOne, Cond } from "components/Conditionals/ChooseOne";
+import { EmptyState } from "components/EmptyState/EmptyState";
+import { Paywall } from "components/Paywall/Paywall";
+import { Stack } from "components/Stack/Stack";
+import { StatusIndicator } from "components/StatusIndicator/StatusIndicator";
+import {
+	TableLoaderSkeleton,
+	TableRowSkeleton,
+} from "components/TableLoader/TableLoader";
+import type { FC } from "react";
+import { docs } from "utils/docs";
+
+export type IdpSyncPageViewProps = {
+	oidcConfig: OIDCConfig | undefined;
+};
```

Comment:
I actually don't know yet what the API response will look like. All I know is that it will have some similarity to what exists in OIDCConfig. So right now this is just meant to be an approximation to mock some data with the assumption that it will change.